### ISSUE TITLE
Header expands when opening modal on the same screen.

### ISSUE THREAD

**Issue by azwinlam** [PedroBern/react-native-collapsible-tab-view#269]
When scrolling through the flatlist, the header collapse as per expected behavior. When I onPress an item that opens a modal.

The header then expands to it's original size and the flatlist scrolls back to the first item.

Anyone experience this behavior?

**Comment by Ashutosh-Azilen** [comment]
Yes, this is happening for me when I open keyboard on search input click (which I have passed as header component of the Tabs.SectionList.
